src/klein/_decorators.py

```diff
@@ -20,11 +24,16 @@ def bindable(bindable):
     @return: its argument, modified to mark it as unconditinally requiring an
         instance argument.
     """
-    bindable.__klein_bound__ = True
+    bindable.__klein_bound__ = True  # type: ignore[attr-defined]
```

Comment:
Longer-term we should probably define a `Protocol` that defines `__klein_bound__` (and `__call__` probably?) so we can document it somewhere, then `cast()` to that rather than `type: ignore`. This is fine for now though.

Reply:
Filed as #344